.prettierrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "semi": false,
```

Comment:
I've opted to exclude semi-colon insertion here since:
 - I think the code looks better without them
 - The existing code doesn't currently use it
If anyone feels strongly that we should require semi-colons, I don't have any issues with removing the rule override.

Reply:
are we doing   [![JavaScript Style Guide](https://cdn.rawgit.com/standard/standard/master/badge.svg)](https://github.com/standard/standard)?  🎊

Reply:
More-or-less, yeah.  The adjustments applied by the `.prettierrc.json` file may not be entirely necessary but I can't seem to get those rules to carry through despite the fact that we're applying the `standard` rules in `.eslintrc.json`.